lib/view/disclaimer_view.dart

```diff
@@ -100,24 +100,29 @@ class _DisclaimerViewState extends State<DisclaimerView> {
                           ' for this.',
                           style: Styles.textH5,
                         ),
-                        Text.rich(TextSpan(children: [
-                          const TextSpan(
-                            text: '\n\nPlease refer to ',
-                            style: Styles.textH5,
-                          ),
+                        Text.rich(
                           TextSpan(
-                              text: whCoronavirusInfoURL,
-                              style: Styles.textH5Hyperlink,
-                              recognizer: TapGestureRecognizer()
-                                ..onTap = () {
-                                  launch(whCoronavirusInfoURL);
-                                }),
-                          const TextSpan(
-                            text:
-                                ' to ensure you have the latest up-to-date information on Western Health\'s Coronavirus guidelines.\n\n',
-                            style: Styles.textH5,
+                            children: [
+                              const TextSpan(
+                                text: '\n\nPlease refer to ',
```

Comment:
better to use padding/margin vs embedded newlines. also shouldn't this be in string consts?

Reply:
Yeah sure.

Reply:
Not part of this PR, so so later in another PR when Disclaimer content issues are resolved.